<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>met4</class>
 <widget class="QMainWindow" name="met4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>met4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Matriz 3x3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_13"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>X1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_14"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>X2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_15"/>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>X3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>X1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>X2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>X3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_9"/>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>X1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_10"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>X2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_11"/>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>X3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>METODO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>BORRA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>SAL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_6"/>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>X1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_5"/>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>X2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_7"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>X3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
